CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 cmake_minimum_required(VERSION 3.0.0)
-project("Lily")
+project("Lily" C)
 
 if(MSVC)
     add_definitions(/W4)
     add_definitions(/wd4201)
     add_definitions(/wd4204)
     add_definitions(/wd4214)
     add_definitions(/MP)
+    set(CMAKE_CTEST_COMMAND	copy lily.exe ..)
 else()
+    set(CMAKE_CTEST_COMMAND	cp lily ..)
```

Comment:
No. The cmake build dir name is user-controlled. It can be any name.
Here the cmake generated makefile has to step out of it to run the tests.